pycode/memilio-generation/memilio/generation/template/template_string.py

```diff
@@ -0,0 +1,269 @@
+#############################################################################
+# Copyright (C) 2020-2022 German Aerospace Center (DLR-SC)
+#
+# Authors: Maximilian Betz
+#
+# Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#############################################################################
+"""
+@file template_string.py
+@brief Generate small pieces of the target code as strings.
+"""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from memilio.generation import IntermediateRepresentation
+
+
+def includes(intermed_repr: IntermediateRepresentation) -> str:
+    """
+    @param intermed_repr Dataclass holding the model features.
+    @return Formatted string representing a part of the bindings.
+    """
+    substition_string = (
+        "#include \"pybind_util.h\"\n"
+        "//Includes from pymio\n"
+        "//Includes for the model\n"
+        "//Includes from Memilio\n"
```

Comment:
These strings do not get substituted at the moment.

Reply:
Thats intended for now. I opened the Issue #564 to improve this in the future.